data-non-standard.js

```diff
@@ -1218,7 +1212,7 @@ exports.tests = [
     firefox12: false,
     firefox16: true,
     firefox17: false,
-    firefox20: true,
+    firefox20: false,
     firefox28: false,
```

Comment:
> But for non-standard features table, which is the one we are talking about here (data-non-standard.js), no browsers are marked as "very obsolete".

Wow. Didn't notice. They should match obsolete browsers for other tables IMHO. Did someone had any counterarguments?